proto/greenfield/sp/types.proto

```diff
@@ -29,6 +29,7 @@ enum Status {
   STATUS_IN_JAILED = 1;
   STATUS_GRACEFUL_EXITING = 2;
   STATUS_IN_MAINTENANCE = 3;
+  STATUS_FORCE_EXITING = 4;
```

Comment:
Update challenge module to make sure that sp in exiting status can still be challenged. Two places: 1) submit challenge tx, 2) endblock.

Reply:
it is allowed, added in these two places.